sdk/communication/communication-common/src/identifierModels.ts

```diff
@@ -59,6 +60,31 @@ export interface MicrosoftTeamsUserIdentifier {
   cloud?: "public" | "dod" | "gcch";
 }
 
+/**
+ * A Microsoft Teams bot.
+ */
+export interface MicrosoftTeamsBotIdentifier {
+  /**
+   * Optional raw id of the Microsoft Teams bot.
+   */
+  rawId?: string;
+
+  /**
+   * Id of the Microsoft Teams bot. If the bot isn't anonymous, the id is the AAD object id of the bot.
```

Comment:
I don't know if there is such a thing as an anonymous bot, so I would shorten this to 
```suggestion
   * Id of the Microsoft Teams bot.
```